docs/api/crash-reporter.md

```diff
@@ -116,18 +116,33 @@ called before `start` is called.
 
 **Note:** This API can only be called from the main process.
 
-### `crashReporter.setExtraParameter(key, value)` _macOS_
+### `crashReporter.addExtraParameter(key, value)` _macOS_
```

Comment:
I think it should stay in the docs until it is removed.
But with the deprecating warning and a note what methods to use instead.